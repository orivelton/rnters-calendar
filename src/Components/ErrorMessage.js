import React from 'react';

const ErrorMessage = ({message}) => <p className="alert-danger" role="alert">{message}</p>; 

export default ErrorMessage;