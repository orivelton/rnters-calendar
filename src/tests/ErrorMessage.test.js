import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from '../Components/ErrorMessage';

test('renders correctly', () => {
  const tree = renderer.create(<ErrorMessage />).toJSON();
  expect(tree).toMatchSnapshot();
});
