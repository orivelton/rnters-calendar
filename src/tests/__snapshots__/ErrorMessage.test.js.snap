// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="alert-danger"
  role="alert"
/>
`;
